attorch/losses.py

```diff
@@ -1,9 +1,12 @@
 import torch.nn as nn
 from torch.nn import functional as F
-from torch.nn.modules.loss import _assert_no_grad, BCELoss
+from torch.nn.modules.loss import BCELoss
```

Comment:
Can you check whether we still need BCELoss? If not, remove it.

Reply:
it's not used so I removed it